src/error-list.cpp

```diff
@@ -207,6 +207,10 @@ bool compiled_error_list::is_present(const char* error_code) const noexcept {
   }
   return present;
 }
+
+bool compiled_error_list::empty() const noexcept {
```

Comment:
This name is misleading. An empty `compiled_error_list` would mean that no error code `is_present`.

I'd call this something like `did_add_error_list` or `number_of_added_error_lists`? (I'm very unsure. =S)